src/data/widgets.js

```diff
@@ -24,28 +24,29 @@ const TestnetWidgets = {
   tosCheck: "one.testnet/widget/TosCheck",
   tosContent: "one.testnet/widget/TosContent",
   wrapper: "one.testnet/widget/DIG.Theme",
+  wrapper: "one.testnet/widget/DIG.Theme",
 };
 
 const MainnetWidgets = {
   image: "mob.near/widget/Image",
-  default: "adminalpha.near/widget/ActivityPage",
+  default: "near/widget/ActivityPage",
   viewSource: "mob.near/widget/WidgetSource",
   widgetMetadataEditor: "mob.near/widget/WidgetMetadataEditor",
   widgetMetadata: "mob.near/widget/WidgetMetadata",
   profileImage: "mob.near/widget/ProfileImage",
   profileName: "patrick.near/widget/ProfileName",
   editorComponentSearch: "mob.near/widget/Editor.ComponentSearch",
   profileInlineBlock: "mob.near/widget/Profile.InlineBlock",
-  componentsPage: "adminalpha.near/widget/ComponentsPage",
-  peoplePage: "adminalpha.near/widget/PeoplePage",
+  componentsPage: "near/widget/ComponentsPage",
+  peoplePage: "near/widget/PeoplePage",
   globalSearchPage: "chaotictempest.near/widget/Search",
-  notificationButton: "adminalpha.near/widget/NotificationButton",
-  profilePage: "adminalpha.near/widget/ProfilePage",
-  componentSummary: "adminalpha.near/widget/ComponentSummary",
-  notificationsPage: "adminalpha.near/widget/NotificationsPage",
-  tosCheck: "adminalpha.near/widget/TosCheck",
+  notificationButton: "near/widget/NotificationButton",
+  profilePage: "near/widget/ProfilePage",
+  componentSummary: "near/widget/ComponentSummary",
+  notificationsPage: "near/widget/NotificationsPage",
+  tosCheck: "near/widget/TosCheck",
   tosContent: "adminalpha.near/widget/TosContent",
```

Comment:
intentionally left as `adminalpha.near` so as not to require re-acceptance by all users. We can switch it the next time a TOS update drops